rskj-core/src/test/java/co/rsk/peg/PegTestUtils.java

```diff
@@ -172,23 +169,16 @@ public static Script createOpReturnScriptForRskWithCustomPayload(int protocolVer
         return ScriptBuilder.createOpReturnScript(payloadBytes);
     }
 
-    public static Address createP2PKHBtcAddress(NetworkParameters networkParameters, int pk) {
-        return BtcECKey.fromPrivate(BigInteger.valueOf(pk)).toAddress(networkParameters);
-    }
-
+    /**
+     * @deprecated Use co.rsk.peg.bitcoin.BitcoinTestUtils#createP2PKHAddress(co.rsk.bitcoinj.core.NetworkParameters, java.lang.String) instead.
+     * Avoid using random values in tests
```

Comment:
Lots Of Random Methods In This Class, We Might Want To Create a Task In The Backlog To Handle That Later

Reply:
Yeah, the plan is to move away from this class and start creating specific utils classes for each particular need